ecosystem/lava-sdk/src/sdk/sdk.ts

```diff
@@ -506,27 +509,31 @@ export class LavaSDK {
       );
     }
     const relayResult = rpcConsumerServer.sendRelay(options);
-    return await relayResult.then((response) => {
-      // // Decode response
-      const reply = response.reply;
-      if (reply == undefined) {
-        throw new Error("empty reply");
-      }
-      const dec = new TextDecoder();
-      const decodedResponse = dec.decode(reply.getData_asU8());
-      // Parse response
-      const jsonResponse = JSON.parse(decodedResponse);
-      // Return response
-      return jsonResponse;
-    });
+    return await relayResult
+      .then((response) => {
+        // // Decode response
+        const reply = response.reply;
+        if (reply == undefined) {
+          throw new Error("empty reply");
+        }
+        const dec = new TextDecoder();
+        const decodedResponse = dec.decode(reply.getData_asU8());
+        // Parse response
+        const jsonResponse = JSON.parse(decodedResponse);
+        // Return response
+        return jsonResponse;
+      })
+      .catch((err) => {
+        console.log(err);
```

Comment:
we need to throw or return just printing is not enough

Reply:
I think it was a log for myself. Removing